src/osx/cocoa/window.mm

```diff
@@ -3024,12 +3024,30 @@ static NSComparisonResult CocoaLowerWindowCompareFunction(KindOfView first, Kind
     }
 }
 
+static void SetSubviewsNeedDisplay( NSView *view )
+{
+    for ( NSView *sub in view.subviews )
```

Comment:
Which macOS/compiler version does this syntax require?

Reply:
Objective-C 2.0, i.e. OS X 10.5 or newer.